dags/common/utils.py

```diff
@@ -228,3 +227,63 @@ def process_archive(file_bytes, file_name):
         return process_zip_file(file_bytes, file_name)
     if tarfile.is_tarfile(file_bytes):
         return process_tar_file(file_bytes, file_name)
+
+
+def process_zip_file_elsevier(file_bytes, file_name):
+    file_bytes.seek(0)
+    with zipfile.ZipFile(file_bytes) as zip:
+        for filename in zip.namelist():
+            if "dataset.xml" not in filename:
```

Comment:
this is the only difference between `process_zip_file_elsevier` and ` process_zip_file` right? why not just add an `ignore_file_name: list` param to the `process_zip_file` instead of code duplication?

Reply:
True! Good idea, thanks!